presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -0,0 +1,249 @@
+
+You must make the following changes to the environment prior to configuring the
+Presto coordinator to use LDAP authentication and HTTPS. 
+
+ * :ref:`ldap_server`
+ * :ref:`server_java_keystore`
+
+You also need to make following changes to the Presto configuration files.
+
+config.properties
+~~~~~~~~~~~~~~~~~
+
+LDAP authentication is configured in the coordinator node's
+config.properties file. Sample entries that need to be added are listed below.
+
+.. code-block:: none
+
+    authentication.ldap.enabled=true
+    authentication.ldap.url=ldaps://ldap-server:636
+
+    http-server.https.enabled=true
+    http-server.https.port=8443
+
+    http-server.https.keystore.path=/etc/presto_keystore.jks
+    http-server.https.keystore.key=keystore_password
+
+In addition to the above properties, you will need to add LDAP server implementation 
+specific properties. 
+
+For Active Directory:
+
+.. code-block:: none
+
+    authentication.ldap.server-type=active_directory
+    authentication.ldap.ad-domain=corp.domain.com
+
+For OpenLDAP:
+
+.. code-block:: none
+
+    authentication.ldap.server-type=openLDAP
+    authentication.ldap.base-dn=OU=Asia,DC=corp,DC=domain,DC=com
+
+======================================================= ======================================================
+Property                                                Description
+======================================================= ======================================================
+``authentication.ldap.enabled``                         Enable LDAP authentication for the Presto coordinator.
+                                                        Must be set to ``true``. Default value is ``false``.
+``authentication.ldap.url``                             The url to the LDAP server. We recommend using 
+                                                        ``ldaps://`` to enable Secure LDAP over SSL.
+``authentication.ldap.server-type``                     The type of LDAP server implementation.
+                                                        Set to ``active_directory`` or ``openLDAP``.                                                        
+``authentication.ldap.ad-domain``                       The domain name of the Active Directory server.
+                                                        This property is required only if 
+                                                        ``authentication.ldap.server-type`` is set to
+                                                        ``Active_Directory``. Eg: ``corp.domain.com``.
+``authentication.ldap.base-dn``                         The base LDAP distinguished name for the user who
+                                                        who tries to connect to the server. This property is 
+                                                        required if ``authentication.ldap.server-type`` is set
+                                                        to ``openLDAP``.
```

Comment:
Should we rather add this clause too: "or when specifying LDAP group membership properties." 
I know I had earlier said that we can remove this, but I am now thinking that the current text doesn't give the complete information.